### DISCUSSION TITLE
Application for technical writer

### DISCUSSION THREAD

**Discussion by ekasnh** [learningequality/kolibri#8089] [Q&A] [answered]
Greetings of the day 
Hi my name is Ekansh Agarwal. I am pursuing Electronics and telecommunication Engineering.I would like to contribute to the project as a technical writer under this year's Google Season of Docs. Requesting your guidance for the same.

Regards
Ekansh Agarwal

**Comment by radinamatic** [comment, marked as answer]
Hi @ekasnh ,

Thank you for your interest in collaborating with Learning Equality in the Google Season of Docs 2021.

We concluded the recruiting process for 2021 year's edition, but we would like to invite you to sign up for periodic email updates from Learning Equality to stay informed about:

* Updates from GSoD writing projects.
* The ongoing impact of the broader Kolibri Product Ecosystem.
* Other opportunities to get involved with Learning Equality.

You can sign-up here: [le.fyi/newsletter](le.fyi/newsletter).
